src/css/layout/header.css

```
@@ -0,0 +1,141 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 8px;
+}
+
+@media (min-width: 768px) {
+  .header-container {
+    margin-bottom: 32px;
+  }
+}
+
+@media (min-width: 1280px) {
+  .header-container {
+    margin-bottom: 16px;
+  }
+}
+
+.logo {
+  display: block;
+  padding: 20px 0;
+  font-weight: 700;
+  font-size: 20px;
+  color: var(--primary-text-color);
+}
+
+.logo-accent {
+  color: var(--primary-brand-color);
+}
+
+.main-navigation {
+  display: none;
+}
+
+@media (min-width: 1280px) {
+  .main-navigation {
+    display: flex;
+    align-items: center;
+  }
+  .site-nav {
+    display: flex;
+    gap: 20px;
+  }
+  .nav-link {
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 1.28571;
+    color: var(--primary-text-color);
+    position: relative;
+  }
+
+  .nav-link:hover:after,
+  .nav-link:focus:after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: -4px;
+    width: 37px;
+    height: 2px;
+    background-color: var(--primary-brand-color);
+    border-radius: 2px;
+  }
+}
+
+.header-social {
+  display: none;
+}
+
+@media (min-width: 1280px) {
+  .header-social {
+    display: flex;
+    gap: 12px;
+  }
+  .header-social-link {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    flex-direction: row;
+    gap: 8px;
+    border: 1px solid rgba(18, 20, 23, 0.1);
+    border-radius: 10px;
+    padding: 8px;
+    width: 44px;
+    height: 44px;
+  }
+
+  .header-social-link:hover,
+  .header-social-link:focus {
+    border: 1px solid rgba(77, 194, 116, 0.1);
+  }
+  .header-social-link .social-icon {
+    width: 28px;
+    height: 28px;
+    fill: rgba(18, 20, 23, 0.8);
+  }
+  .header-social-link .social-icon:hover,
```

Правильно буде перенести ховер на кнопку і змінити для іконок філл, тоді анімацію буде правильною, коли наводиш на кнопку
```suggestion
  .header-social-link:hover .social-icon,
  .header-social-link:focus .social-icon {
    fill: rgba(77, 194, 116, 0.8);
  }
```